<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QComboBox" name="comboBoxAmmount">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>15</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBoxPacksPerDay">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>0.5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBoxCurrency">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>BYN</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>YEN</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RUB</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>USD</string>
       </property>
      </item>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="maximum">
       <double>9.900000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>3.600000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Price per pack</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Packs per day</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>-10</y>
        <width>291</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(0, 0, 0, 255));</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Curr</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Ammo</string>
      </property>
     </widget>
     <zorder>label</zorder>
     <zorder>doubleSpinBox</zorder>
     <zorder>label_3</zorder>
     <zorder>comboBoxAmmount</zorder>
     <zorder>comboBoxPacksPerDay</zorder>
     <zorder>comboBoxCurrency</zorder>
     <zorder>label_2</zorder>
     <zorder>label_4</zorder>
     <zorder>label_5</zorder>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>361</width>
      <height>681</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>359</width>
       <height>679</height>
      </rect>
     </property>
     <widget class="QPushButton" name="pushButtonMove">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>0</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Settings</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonMinus">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>260</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 28pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>â€”</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonPlus">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>260</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 550 36pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelCigsCurrentDay">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>270</y>
        <width>50</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 36pt &quot;Ubuntu&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelTime">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonTotal">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>510</y>
        <width>88</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Total Price</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelBackground">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>350</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelMotivationText">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>410</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;TextLabel&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <zorder>labelBackground</zorder>
     <zorder>pushButtonMove</zorder>
     <zorder>pushButtonMinus</zorder>
     <zorder>pushButtonPlus</zorder>
     <zorder>labelCigsCurrentDay</zorder>
     <zorder>labelTime</zorder>
     <zorder>pushButtonTotal</zorder>
     <zorder>labelMotivationText</zorder>
    </widget>
   </widget>
   <zorder>scrollArea</zorder>
   <zorder>stackedWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
